test/functional/flowcontrol/utils.go

```diff
@@ -0,0 +1,32 @@
+package flowcontrol
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/openshift/cluster-logging-operator/internal/constants"
+	"github.com/openshift/cluster-logging-operator/test/framework/functional"
+)
+
+var (
+	LokiNsQuery     = `{kubernetes_namespace_name=~"%s"}`
+	LokiAuditQuery  = `{log_type="audit"}`
+	AllLogs         = `.+`
+	MessageTemplate = "$(date -u +'%Y-%m-%dT%H:%M:%S.%N%:z') stdout F $msg "
+	FileTemplate    = "%s/%s_%s_%s/%s/0.log"
+	LogPath         = "/var/log/pods"
+)
+
+func WriteApplicationLogs(f *functional.CollectorFunctionalFramework, numOfLogs int) error {
```

Comment:
What is special about these "write" functions that can not be replaced by those in test/framework/functional/write.go

Reply:
This function writes `numOfLogs` lines in 1 sec. In the implementation, we sleep for `1/numOfLogs` secs after every echo. The other "write" functions don't do that, right? 
For these tests we want to maintain a rate at which logs are written so that validate the rate of logs flow through vector.

Reply:
+1 this seems to be identical to 
```
func (f *CollectorFunctionalFramework) WritesApplicationLogs(numOfLogs int) error {
```

Reply:
@alanconway But for this function we won't have a control on the log generation rate, right?

Reply:
@Pranjal-Gupta2 so please add  a new signiture to `test/framework/functional/write.go` in the event we wish to reuse it